packages/hub/src/scope.ts

```diff
@@ -288,6 +293,9 @@ export class Scope implements ScopeInterface {
       this._tags = { ...this._tags, ...captureContext._tags };
       this._extra = { ...this._extra, ...captureContext._extra };
       this._contexts = { ...this._contexts, ...captureContext._contexts };
+      if (captureContext._requestSessionStatus) {
+        this._requestSessionStatus = captureContext._requestSessionStatus;
+      }
```

Comment:
Ugh there's so much repetition going on in this method, the tiny difference being `_` prefixes.

Could you at least keep the symmetry and add the new lines below `fingerprint`, similar to where it is in the other `else if` branch?